doc/source/conf.py

```diff
@@ -10,9 +10,11 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+import MDAnalysis as mda
```

Comment:
This line will require you to create a full-blown installation (probably via conda) to install MDAnalysis.

Running conda takes about 6 minutes on Travis CI, most of it when it resolves the dependency graph. Perhaps one can speed up by providing a frozen requirements file (or whatever the conda equivalent is).

Reply:
This is for the API autosummary of code. Possibly that should be done in the MDAnalysis repo itself, like the current docs?
(That's also why there are a million files in this pull request, sorry)

Reply:
Yes, if it is generated from the docs then it should go into the standard MDA docs. 

Are these API autosummaries crucial for the user guide right now? If yes, then you either need to push hard to get them into MDA develop or make the CI for the UserGuide work.

I think you can use inter-sphinx links to automatically link to the standard docs.

Reply:
I was arranging the API reference into sections to organise my thoughts (e.g. below) but it makes far more sense to do this in the MDA repo.
<img width="836" alt="Screenshot 2019-10-10 at 9 33 32 AM" src="https://user-images.githubusercontent.com/31115101/66583146-95c4be00-ebce-11e9-81af-6266dcce7cc4.png">